### PR TITLE
WIP: Create a simple rsync command

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

File: src/app/FakeLib/RSyncHelper.fs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿[<AutoOpen>]
+/// Contains a task to use [rsync](https://rsync.samba.org/) on Linux.
+module Fake.RSync
+
+/// Executes a rsync command.
+/// ## Parameters
+///  - `source` - The source directory
+///  - `destination` - The target directory
+let RSync (source:string) (destination:string) =
```
Comment (mexx) [inline src/app/FakeLib/RSyncHelper.fs]: Shouldn't the order of parameters be `destination` `source` for better composability?

**Comment by forki** [comment]
sure, but I copied it from XCopy

File: src/app/FakeLib/RSyncHelper.fs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿[<AutoOpen>]
```
Comment (mexx) [inline src/app/FakeLib/RSyncHelper.fs]: Did you changed your mind about `AutoOpen`?

Comment (forki) [inline src/app/FakeLib/RSyncHelper.fs]: Nope will remove it.

**Comment by forki** [comment]
According to @jchannon this throws with exit code 23 for him. Probably because it runs in elavated user priveledges.

**Comment by matthid** [comment]
Re-Open to track if we want to have this eventually or remove the branch.

**Pull request closed without merging**